eq-author/src/App/dataSettings/ONSDataset/index.js

```diff
@@ -139,171 +154,204 @@ const ONSDatasetPage = () => {
+                                                  type="button"
+                                                  variant="secondary"
+                                                  data-test="btn-link"
+                                                >
+                                                  Link
+                                                </StyledButton>
+                                              </SpacedTableColumn>
+                                            </TableRow>
+                                          );
+                                        }
+                                      )}
+                                    </StyledTableBody>
+                                  )}
+                                </Table>
+                              </>
+                            )}
+                          </>
+                        )}
+                        {tableData && (
+                          <>
+                            <Grid>
+                              <Column gutters={false} cols={10}>
+                                <Title>
+                                  Dataset for survey ID {tableData.surveyId}
+                                </Title>
+                              </Column>
+                              <Column gutters={false} cols={2}>
+                                <UnlinkButton
+                                  data-test="btn-unlink-dataset"
+                                  onClick={handleUnlinkClick}
+                                >
+                                  Unlink dataset
+                                </UnlinkButton>
+                              </Column>
+                            </Grid>
+                            <CustomGrid>
+                              <Column gutters={false} cols={2}>
+                                ID:
+                              </Column>
+                              <CustomColumn gutters={false} cols={2}>
+                                {tableData.id}
+                              </CustomColumn>
+                            </CustomGrid>
+                            <CustomGrid>
+                              <Column gutters={false} cols={2}>
+                                Version:
+                              </Column>
+                              <CustomColumn gutters={false} cols={2}>
+                                {tableData.version}
+                              </CustomColumn>
+                            </CustomGrid>
+                            <CustomGrid>
+                              <Column gutters={false} cols={2}>
+                                Date created:
+                              </Column>
+                              <CustomColumn gutters={false} cols={2}>
+                                {formatDate(tableData.dateCreated)}
+                              </CustomColumn>
+                            </CustomGrid>
+                          </>
```

Comment:
This doesn't include the rest of the text for the page as in the Jira ticket attached Figma. I know there are issues with the table output, but shouldn't it include the rest of the text here?

Reply:
@Paul-Joel Good question - I should have mentioned in the review instructions, had a discussion with Martyn, since the data fields will be done in the next ticket https://jira.ons.gov.uk/browse/EAR-2130, the static text will also be taken care in that ticket. Hope this helps.

Reply:
@Paul-Joel Just included the Figma design link in the ticket for clarity - https://jira.ons.gov.uk/browse/EAR-2130